tests/console/journalctl.pm

```diff
@@ -148,15 +148,15 @@ sub run {
         die "journal lists less than 2 boots" if (scalar(@listed_boots) < 2);
         is_journal_empty('--boot=-1', "journalctl-1.txt");
         assert_script_run('journalctl --identifier=batman --boot=-1| grep "The batman is going to sleep"', fail_message => "Error getting beacon from previous boot");
+        script_run('echo -e "Reboot time:  `cat /var/tmp/reboottime`\nCurrent time: `date -u \'+%F %T\'`"');
```

Comment:
extra whitespace or is it intentional?
Also would that make sense to be as a record_info in the test?

Reply:
The line seems identical. :)
`record_info` would have the same result as the test case uses serial terminal